src/main/java/storage/Storage.java

```diff
@@ -126,11 +96,35 @@ public AuthorList readAuthorListFromDataFile() {
             logger.info("Data restored");
             return authorList;
         } catch (IOException e) {
-            logger.warning("Problems reading file, data was not restored!" + e.getMessage());
+            logger.warning("Problems accessing file, data was not restored!" + e.getMessage());
+        } catch (JsonParseException e) {
+            logger.warning("Problems parsing JSON from file, data was not restored!" + e.getMessage());
```

Comment:
Might consider throwing exceptions and `sout` them to the user. Not sure if logging is something that the user should see. To be discussed.

Reply:
Good suggestion, will implement soon.